src/pydata_sphinx_theme/assets/styles/sections/_announcement.scss

```diff
@@ -53,9 +53,13 @@
     border-radius: 0.25rem;
 
     &:hover {
+      // applying the same style as the SD "buttons"
+      @include link-style-hover;
+
+      text-decoration-thickness: 1px;
```

Comment:
Seems this was missed, so I added this to bring parity with other buttons @gabalafou  unless there was a reason to leave this out?

Reply:
Correct. Plus, I like this parity because the text underline suggests that the button is actually a link, which is the case.